### ISSUE TITLE
[DOC] Include object/string repesentation comparison in STASH code example

### ISSUE THREAD

**Issue by kaedonkers** [SciTools/iris#3063]
A suggestion from @nhsavage:

It would be useful to have an example of comparing the stash object to its string representation in the documentation:
https://scitools.org.uk/iris/docs/latest/iris/iris/fileformats/pp.html?highlight=iris%20fileformats%20pp%20stash#iris.fileformats.pp.STASH

This would be very useful but is currently not obvious, IMHO.

e.g.
```
>>> cube=iris.load_cube('sample_theta.pp')
>>> cube.attributes.get('STASH')
STASH(model=1, section=0, item=4)
>>> cube.attributes.get('STASH') == 'm01s0i004'
True
```

**Comment by pelson** [comment]
Good idea. I'd want the example to be self contained, and not involve cubes though. So just:

```
>>> STASH(model=1, section=0, item=4) == 'm01s0i004'
True
```

**Comment by pelson** [comment]
Happy to submit a PR against ``v2.1.x`` @kaedonkers / @nhsavage? Happy to lend a hand if necessary.

**Comment by nhsavage** [comment]
I need to wrap my head around git/github and so I'd like to give it a go

**Comment by pelson** [comment]
Closed by #3072. Will be in v2.1.1 or v2.2.0, whichever comes first.

Thanks for doing this! 👍
